apps/pm-user/src/components/molecules/loginForm/LoginForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import styles from './loginForm.module.scss';
+import { Button, Input } from '../..';
+
+const LoginForm = ({ onLogin }) => {
+  const [formValues, setFormValues] = useState({
+    emailValue: '',
+    passwordValue: '',
+  });
+
+  function submitForm(e) {
+    e.preventDefault();
+
+    if (formValues) {
+      onLogin(formValues.passwordValue, formValues.emailValue);
+    } else {
+      console.log('email or password can not be empty');
+    }
+  }
+
+  function getFormValues(e) {
+    const value = e.target.value;
+    const name = e.target.name;
+
+    setFormValues((oldVlaues) => {
+      return { ...oldVlaues, [name]: value };
+    });
+  }
+
+  return (
+    <div className={styles.super_container}>
+      <div className={styles.left_container}></div>
+      <div className={styles.right_container}>
+        <div className={styles.login_box}>
+          <div className={styles.login_logo}>Pesto Matrimony</div>
+          <div className={styles.signin_text}>Sign in</div>
+          <form onSubmit={submitForm}>
+            <Input
+              type="email"
+              placeholder="Email Address"
+              name="emailValue"
+              value={formValues.emailValue}
+              onChangeHandler={getFormValues}
```

Comment:
Wouldn't it be better if we say "onChange" for the prop instead of onChangeHandler ?

onChangeHandler is used for naming handler functions. and hence it appears odd.